legacy/oled.c

```
@@ -252,33 +256,45 @@ void oledRefresh() {
 
 void oledBufferLoad(uint8_t *buffer) {
-  memcpy(buffer, _oledbuffer, OLED_BUFSIZE);
+  memcpy(buffer, current_buffer, OLED_BUFSIZE);
 }
 
 void oledBufferRestore(uint8_t *buffer) {
-  memcpy(_oledbuffer, buffer, OLED_BUFSIZE);
+  memcpy(current_buffer, buffer, OLED_BUFSIZE);
+}
+
+void oledSwitchToOverlayBuffer(void) {
```

Cppcheck (2.19.0)

[style] The function 'oledSwitchToOverlayBuffer' is never used.

(unusedFunction)